airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3393,6 +3393,7 @@ components:
         - destinationId
         - syncCatalog
         - status
+        - isBreaking
```

Comment:
I should probably change this to breakingChange, since that's the name of the db field this reflects.